### PR TITLE
Remove armor before LR

### PR DESCRIPTION
**Pull request by Graffiti007**

### CODE REVIEW

**Comment by Graffiti007** [comment]
Added new CVAR Remove armor before LR

**Comment by data-bomb** [comment]
Is potential CT armor causing some fairness issues during an LR?

**Comment by Graffiti007** [comment]
Yes

File: addons/sourcemod/scripting/hosties/lastrequest.sp [hunk 1]
```diff
@@ -551,6 +553,8 @@ LastRequest_OnPluginStart()
 	gShadow_LR_BlockSuicide = false;
 	gH_Cvar_LR_VictorPoints = CreateConVar("sm_hosties_lr_victorpoints", "1", "Amount of frags to reward victor in an LR where other player automatically dies", 0, true, 0.0);
 	gShadow_LR_VictorPoints = 1;
+	gH_Cvar_LR_RemoveArmor = CreateConVar("sm_hosties_lr_removearmovr", "1", "Remove armor before LR", 1, true, 0.0);
```
Comment (data-bomb) [inline addons/sourcemod/scripting/hosties/lastrequest.sp]: sm_hosties_lr_removearmovr should be sm_hosties_lr_removearmor

Comment (data-bomb) [inline addons/sourcemod/scripting/hosties/lastrequest.sp]: Why is this a 1 and all others are 0 for the convar flags?  Originally these were all FCVAR_PLUGIN.  I'm not sure why we changed to magic numbers.

File: addons/sourcemod/scripting/hosties/lastrequest.sp [hunk 2]
```diff
@@ -3652,6 +3662,13 @@ InitializeGame(iPartnersIndex)
 		LogToGame("\"%L\" started a LR game (\"%s\") with \"%L\"", LR_Player_Prisoner, LR_Name, LR_Player_Guard);
 	}
 	
+	//remove armor
+	if (gShadow_LR_RemoveArmor == 1)
```
Comment (data-bomb) [inline addons/sourcemod/scripting/hosties/lastrequest.sp]: When you CreateConVar'd you did not specify a maximum, so you should only check if the value is positive.  Change it to
if (gShadow_LR_RemoveArmor)
to account for this.

File: addons/sourcemod/scripting/hosties/lastrequest.sp (hunk 1 above)
Comment (Bara20) [inline addons/sourcemod/scripting/hosties/lastrequest.sp]: btw. FCVAR_PLUGIN is with SM1.8+ deprecated. ( #5  )

Comment (Bara20) [inline addons/sourcemod/scripting/hosties/lastrequest.sp]: and set max. value to "true, 1.0" (it's a bool)

**Comment by DorCoMaNdO** [comment]
I think that it'd be beneficial to change all the cvar flags from numbers to the proper definition, it seems like it causes confusion at the current state (seeing that it was assumed to be 1 here, which is wrong). Just because FCVAR_PLUGIN was never used correctly and was deprecated doesn't mean there's no alternative, FCVAR_NONE would do it, this should have been done in the pull request that changed it.

**Comment by data-bomb** [comment]
Agreed, will change to FCVAR_NONE here and have a follow-on for the rest of the convars.

**Pull request merged**